exercises/dnd-character/DndCharacterTest.cs

```diff
@@ -140,4 +140,36 @@ public void Each_ability_is_only_calculated_once()
             Assert.Equal(sut.Charisma, sut.Charisma);
         }
     }
-}
\ No newline at end of file
+    
+    [Fact(Skip = "Remove to run test")]
+    public void Random_ability_is_distributed_correctly()
+    {
+        var idealDistribution = new Dictionary<int, int>() {
+            [3] = 1,            [4] = 4,
```

Comment:
Any reason for the large number of spaces between the elements? Personally, I think having a little less spaces might be a bit easier to read:

```suggestion
            [3] = 1,        [4] = 4,
```

Reply:
Ok